Atomic/syscontainers.py

```diff
@@ -2239,7 +2235,8 @@ def _check_system_oci_image(self, repo, img, upgrade, src_creds=None):
 
         if can_use_skopeo_copy:
             return self._check_system_oci_image_skopeo_copy(repo, img, src_creds=src_creds)
-        return self._check_system_oci_image_no_skopeo_copy(repo, img, imagebranch)
+        else:
+            raise ValueError("Skopeo version too old, please install the newest version")
```

Comment:
**Question**: Do we want to have a hard dependency on skopeo version here? We can also use Makefile to enforce the version match (>= a specific version of skopeo).  

We can also choose to give user a warning instead of raising errors, i am open to suggestions


Another doubt here is that, how do we retrieve the version of the "working" skopeo, so users know which version at least they need to install? The dilemma here is to decide between whether we should hard code version here, or we retrieve it dynamically (dunno how to do it tho)

Reply:
I don't think we can use Makefile to ensure the version of skopeo as where a build happens and where a tool is installed will likely be different hosts. We can add a dependency check in the RPM spec file. If we want to make a hard need on version a util function that checks the skopeo version on first usage could be made but that's a different PR.

Reply:
Thanks for the comment :). 

>If we want to make a hard need on version a util function that checks the skopeo version on first usage could be made but that's a different PR.

I think this approach sounds nicer. I also don't think it would be too much work for doing this check as well. Will write up a PR to check for skopeo version after this PR.

Reply:
yes I agree with @ashcrow, this problem must be solved in the RPM .spec file.

If you'd still like to raise a nicer error message, you could parse the `skopeo --version` output.

Reply:
Sounds good. /me will spend time learning how to include files in RPM .spec, and push out a PR once figured out how.